presto-lark-sheets/pom.xml

```diff
@@ -16,6 +16,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
```

Comment:
Why was this change needed? Is it because of the change in Presto SPI? 

nit: Let's move this down at the start of all provided scope dependencies.

Reply:
Yes due to the change in presto-spi, I had to add the dependency .